Userland/Applications/FileManager/main.cpp

```diff
@@ -1003,17 +1007,21 @@ ErrorOr<int> run_in_windowed_mode(ByteString const& initial_location, ByteString
         refresh_tree_view();
     });
 
+    auto quit_action = GUI::Action::create("&Quit", { Mod_Alt, Key_F4 }, { Mod_Ctrl, Key_W }, [&](GUI::Action const&) {
+        GUI::Application::the()->quit();
+    });
+    quit_action->set_status_tip("Quit the application"_string);
+
     auto file_menu = window->add_menu("&File"_string);
+    file_menu->add_action(new_window_action);
     file_menu->add_action(mkdir_action);
     file_menu->add_action(touch_action);
     file_menu->add_action(focus_dependent_delete_action);
     file_menu->add_action(directory_view->rename_action());
     file_menu->add_separator();
     file_menu->add_action(properties_action);
     file_menu->add_separator();
-    file_menu->add_action(GUI::CommonActions::make_quit_action([](auto&) {
```

Comment:
We have a dedicated `GUI::CommonActions::make_quit_action` for quit. It seems weird to not use that. `make_quit_action` uses alt+f4. It seems weird to have apps differ in their "close window" shortcuts.

PixelPaint uses ctrl-w to close the current tab, alt-f4 to close the current window. So I'm not sure if we can just add ctrl-w to make_quit_action.

Reply:
> We have a dedicated `GUI::CommonActions::make_quit_action` for quit. It seems weird to not use that.

I did use that originally. Then it was suggested that Ctrl-W should be a secondary shortcut for quit instead, rather than adding a separate  "Close Window" action, and `make_quit_action` does not support secondary shortcuts.

> I'm not sure if we can just add ctrl-w to make_quit_action.

I can revert to the original behavior of adding a separate "Close Window" action if that would be more acceptable.

Reply:
I agree that alt+f4 is the universal shortcut to close an app. But ctrl+w is quite popular as well (I personally use it a lot), it closes tabs first but also close the app when only one tab remains.
Maybe we can make it an opt-in when calling `make_quit_action`? This way apps with tabs can still manage the shortcut themselves and others can opt in.

Reply:
> I did use that originally. Then it was suggested that Ctrl-W should be a secondary shortcut for quit instead, rather than adding a separate "Close Window" action, and make_quit_action does not support secondary shortcuts.

I think my main point is that it's weird that ctrl-w now works in FileManager but not in most other apps. "Close thing" isn't app-specific; it should behave the same across most apps.

Giving `make_quit_action` a (default-true) toggle to close the current window and disabling apps using tabs (pixelpaint, browser) and in apps using ctrl-w for other things (Terminal) sounds like a good idea to me.

Reply:
> Giving `make_quit_action` a (default-true) toggle to close the current window and disabling apps using tabs (pixelpaint, browser) and in apps using ctrl-w for other things (Terminal) sounds like a good idea to me.

Ok, I have updated this PR to remove the close window Ctrl-W shortcut, so that a separate PR can deal with adding this feature to `make_quit_action`.

Reply:
That works too!